automatic/mpc-hc-clsid2/update.ps1

```diff
@@ -22,24 +22,24 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
+  $releasesPage = Invoke-WebRequest -Uri $releases -UseBasicParsing
 
-  $re = 'x86\.exe$'
-  $url32 = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
+  $re = "^((?:\d+\.){2}\d+)$"
 
+  $versionHtml = $releasesPage.Links | ? title -match $re | select -first 1
+  $version = $versionHtml.title
+
+  $download_page = Invoke-WebRequest -Uri "$($releases)/tag/$($version)"
```

Comment:
The time out is being caused by the lack of using the `-UseBasicParsing` switch.

Reply:
Thank you. I just pushed a commit to add that.